src/cli/commands/force/lightning/local/device/start.ts

```diff
@@ -0,0 +1,199 @@
+Messages.importMessagesDirectory(__dirname);
+
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is using the messages framework can also be loaded this way.
+const messages = Messages.loadMessages(
+    '@salesforce/lwc-dev-mobile',
+    'device-start'
+);
+
+const LWC_DEV_MOBILE = 'lwc-dev-mobile';
+const DEVICE_START = 'Device Start';
+
+export class Start extends Setup {
+    public static description = messages.getMessage('commandDescription');
+
+    public static readonly flagsConfig: FlagsConfig = {
+        platform: flags.string({
+            char: 'p',
+            description: messages.getMessage('platformFlagDescription'),
+            required: true
+        }),
+        target: flags.string({
+            char: 't',
+            description: messages.getMessage('targetFlagDescription'),
+            required: true
+        }),
+        writablesystem: flags.boolean({
+            char: 'w',
+            description: messages.getMessage('writablesystemFlagDescription'),
+            required: false,
+            default: false
+        })
+    };
+
+    public examples = [
+        `sfdx force:lightning:local:device:start -p iOS -t MySimulator`,
+        `sfdx force:lightning:local:device:create -p Android -t MyEmulator -w`
+    ];
+
+    private platform = '';
+    private target = '';
+    private writableSystem = false;
+
+    public async run(direct: boolean = false): Promise<any> {
+        if (direct) {
+            await this.init(); // ensure init first
+        }
+
+        this.logger.info(
+            `Device Start command invoked for ${this.flags.platform}`
+        );
+
+        return super
+            .run(direct) // validate input parameters + setup requirements
+            .then(() => {
+                // execute the create command
+                this.logger.info(
+                    'Setup requirements met, continuing with Device Start'
+                );
+                return this.executeDeviceStart();
+            });
+    }
+
+    protected async validateInputParameters(): Promise<void> {
+        return super.validateInputParameters().then(() => {
+            const target = this.flags.target as string;
+            const writableSystem = this.flags.writablesystem as boolean;
+
+            // ensure that thetarget flag value is valid
+            if (target == null || target.trim() === '') {
+                return Promise.reject(
+                    new SfdxError(
+                        messages.getMessage(
+                            'error:invalidTargetFlagsDescription'
+                        ),
+                        LWC_DEV_MOBILE,
+                        this.examples
+                    )
+                );
+            }
+
+            this.platform = this.flags.platform;
+            this.target = target;
+            this.writableSystem = writableSystem;
```

Comment:
Might be worth warning somewhere in here, if the platform is iOS, to the effect of "this option will be ignored."

Reply:
...unless that's a hassle to figure out when it's set vs. not (since it has a default value). It's not worthy of a lot of effort. 🙂